app/models/user.rb

```diff
@@ -1,4 +1,12 @@
 class User < ApplicationRecord
-  validates :user_name, presence: true
+  validates :name, presence: true
   validates :photo, presence: true
+
+  has_many :posts
```

Comment:
# Optional
- [ ]  I would advise you to add `author_id ` to the table, to create associations between models because the requirements says: "**Remember that author_id column in posts table should be the foreign_key for the users table**"
👉  `has_many :posts, foreign_key: 'author_id'`